2-mandatory/2-financial-times.js

```diff
@@ -4,36 +4,90 @@
     The home page of the web site has a headline section, which only has space for article titles which are 65 characters or less.
     Implement the function below, which will return a new array containing only article titles which will fit.
 */
-function potentialHeadlines(allArticleTitles) {
-    // TODO
-}
 
+    function potentialHeadlines(allArticleTitles) {
+        // Create an empty array to store the potential headlines
+        const potentialHeadlines = [];
+      
+        // Loop through each article title in the input array
+        for (let i = 0; i < allArticleTitles.length; i++) {
+          const articleTitle = allArticleTitles[i];
+      
+          // Check if the title is 65 characters or less
+          if (articleTitle.length <= 65) {
+            // If it is, add it to the potential headlines array
+            potentialHeadlines.push(articleTitle);
+          }
+        }
+      
+        // Return the array of potential headlines
+        return potentialHeadlines;
+      }
+      
 /*
     The editor of the FT likes short headlines with only a few words!
     Implement the function below, which returns the title with the fewest words.
-    (you can assume words will always be seperated by a space)
+    (you can assume words will always be separated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
```

Comment:
It looks like this solution will give you the title with the fewest characters, but this might not be the fewest words.
With a couple of small changes you could fix this 😄